scenarioo-client/app/build/use-cases-overview/use-cases-overview.component.css

```diff
@@ -23,9 +23,13 @@ sc-usecases-overview .table thead span {
     cursor: pointer;
 }
 
-sc-usecases-overview .table tbody>tr:hover {
+sc-usecases-overview .table tbody>tr:hover td {
```

Comment:
I think it would be cool to encapsulate our tables in our own Filterabel-Table component - such that in future we do not have to duplictae these styles, and logic and so on for tables on serveral pages. Maybe a good exercise for next few weeks for a more complex custom component?